sdk/nodejs/ec2/getNatGateways.ts

```diff
@@ -9,6 +9,26 @@ import * as utilities from "../utilities";
 
 /**
  * This resource can be useful for getting back a list of NAT gateway ids to be referenced elsewhere.
+ *
+ * ## Example Usage
+ *
+ * The following returns all NAT gateways in a specified VPC that are marked as available
+ *
+ * ```typescript
+ * import * as pulumi from "@pulumi/pulumi";
+ * import * as aws from "@pulumi/aws";
+ *
+ * const ngws = aws.ec2.getNatGateways({
+ *     vpcId: vpcId,
+ *     filters: [{
+ *         name: "state",
+ *         values: ["available"],
+ *     }],
+ * });
+ * const ngw = .map(__index => (aws.ec2.getNatGateway({
```

Comment:
The .map does not look right.

Reply:
This is https://github.com/pulumi/pulumi/issues/15859 which is a known issue, linked it to this failure but not blocking on it.